products/mariadb.md

```diff
@@ -45,21 +46,30 @@ auto:
       -   ^mariadb-(?P<major>10)\.(?P<minor>10)\.(?P<patch>([2-9]|\d{2}))$
       -   ^mariadb-(?P<major>10)\.(?P<minor>11)\.(?P<patch>([2-9]|\d{2}))$
       -   ^mariadb-(?P<major>11)\.(?P<minor>[0-7])\.(?P<patch>([2-9]|\d{2}))$
+  -   release_table: https://mariadb.org/about/#maintenance-policy
+      selector: "table"
+      header_selector: "tbody tr:nth-of-type(1)"
+      ignore_empty_releases: true # Ignore future releases
+      fields:
+        releaseCycle: "Release"
+        releaseDate: "GA release date"
+        eol: "Community"
+        eoes: "Extended"
   -   release_table: https://mariadb.org/about/#maintenance-policy
       selector: "table"
       header_selector: "tr:nth-of-type(1)"
       fields:
-        releaseCycle: "Release series"
-        releaseDate: "Stable (GA) Date"
-        eol: "End-of-life"
+        releaseCycle: "Release"
+        releaseDate: "GA release date"
+        eol: "Community"
 
-# Extend/update regex in section above
+# When adding new Major, do no forget to review regexes in section above.
 releases:
 -   releaseCycle: "11.7"
-    releaseDate: 2025-02-13
-    eol: false
+    releaseDate: 2025-02-12
```

Comment:
```suggestion
    releaseDate: 2024-09-25
```
This sould be 11.7's cycle's first release date not 11.7.2's date so : 
![image](https://github.com/user-attachments/assets/9f66982a-3329-47cc-9aa9-fa92cf73f442)
[ from https://mariadb.com/kb/en/mariadb-11-7-0-release-notes/ ]

Reply:
11.7.0 is an alpha release, see https://mariadb.org/about/#maintenance-policy for ga dates.